app/Http/Controllers/PostTranslationController.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+
+use App\Models\PostTranslation;
+
+class PostTranslationController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        //
+    }
+
+    /**
+     * Show the form for creating a new resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function create()
+    {
+        //
+    }
+
+    /**
+     * Store a newly created resource in storage.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @return \Illuminate\Http\Response
+     */
+    public function store(Request $request)
+    {
+        //
+    }
+
+    /**
+     * Display the specified resource.
+     *
+     * @param  int  $id
+     * @return \Illuminate\Http\Response
+     */
+    public function show($id)
+    {
+        //
+    }
+
+    /**
+     * Show the form for editing the specified resource.
+     *
+     * @param  int  $id
+     * @return \Illuminate\Http\Response
+     */
+    public function edit($id)
+    {
+        $postTranslation = PostTranslation::find($id);
+
+        return view('posts.translations.edit', ['postTranslation' => $postTranslation]);
+    }
+
+    /**
+     * Update the specified resource in storage.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @param  int  $id
+     * @return \Illuminate\Http\Response
+     */
+    public function update(Request $request, $id)
+    {
+        $postTranslation = PostTranslation::find($id);
+        $postTranslation->title = $request->title;
+        $postTranslation->content = $request->content;
+        $postTranslation->description = $request->description;
+        $postTranslation->save();
+
+        return redirect()->route('posts.index');
+    }
+
+    /**
+     * Remove the specified resource from storage.
+     *
+     * @param  int  $id
+     * @return \Illuminate\Http\Response
+     */
+    public function destroy($id)
```

Comment:
validate id

Reply:
Ở đây dùng cách nào để validate mà ko phải query ko
Ở đây mình dùng: Post::findOrFail($id)->delete();

Reply:
muốn validate id xem có trong DB ko thì bắt buộc phải query, ở đây an toàn nhất bạn dùng Model binding biến $id được inject vào thành model bạn nhé